Mathlib/Data/Bitvec/Defs.lean

```diff
@@ -1,339 +1,143 @@
 /-
-Copyright (c) 2015 Joe Hendrix. All rights reserved.
+Copyright (c) 2023 Wojciech Nawrocki. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Joe Hendrix, Sebastian Ullrich
+Authors: Harun Khan, Alex Keizer, Abdalrhman M Mohamed
 -/
```

Comment:
Why does the copyright mention a non-author?

Reply:
This code is an import from @mhk119 and his collegues at Stanford. AFAIU Abdalrhman co-authored some of the pieces, but is just not involved in upstreaming. @mhk119, can you clarify?

Reply:
I think Eric is referring to the fact that Wojciech Nawrocki is listed as the copyright-holder, but not as an author. I think that's because the original PR adapted code by Wojciech in a different project. However, I suspect that all of this person's code has now been upstreamed to Std, so the copyright attribution might no longer make sense for this PR in the current form

Reply:
Yes, exactly; it's @Vtec234's mention in the copyright but not the authors that confuses me.

Reply:
I see. So I guess we either add Wojciech to the authors or add somebody else to the copyright. @mhk119, what would you suggest?

Reply:
Frankly I would just leave the original 2015 copyright and *maybe* add the three new authors. Given the file is mostly aligns, I don't think there's much to claim authorship of anyway.

Reply:
That is a good idea. I changed the headers accordingly.

Reply:
Hi, sorry for the late reply. Daytime in the UK is nighttime in California :). Sure, @eric-wieser's suggestion sounds good. Thanks!